.travis.yml

```diff
@@ -1,4 +1,16 @@
 language: python
 python: "2.7"
-install: "pip install -r requirements.txt"
-script: py.test
+
+addons:
+  apt:
+    packages:
+      - oracle-java8-installer
+      - oracle-java8-set-default
+
+install:
+  - pip install -r requirements.txt
+  - (cd Applications/SimpleUI; npm install)
```

Comment:
OK that's pretty neat. This automatically installs npm grunt modules? I presume we've already installed node.js somewhere?